specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -2382,10 +2570,184 @@
+          "JPBankAccountNumber",
+          "JPDriversLicenseNumber",
+          "JPPassportNumber",
+          "JPResidentRegistrationNumber",
+          "JPSocialInsuranceNumber",
+          "JPMyNumberCorporate",
+          "JPMyNumberPersonal",
+          "JPResidenceCardNumber",
+          "LVPersonalCode",
+          "LTPersonalCode",
+          "LUNationalIdentificationNumberNatural",
+          "LUNationalIdentificationNumberNonNatural",
+          "MYIdentityCardNumber",
+          "MAIdentityCardNumber",
+          "MTTaxIDNumber",
+          "NLCitizensServiceNumber",
+          "NLCitizensServiceNumberV2",
+          "NLTaxIdentificationNumber",
+          "NLValueAddedTaxNumber",
+          "NZBankAccountNumber",
+          "NZDriversLicenseNumber",
+          "NZInlandRevenueNumber",
+          "NZMinistryOfHealthNumber",
+          "NZSocialWelfareNumber",
+          "NOIdentityNumber",
+          "PHUnifiedMultiPurposeIDNumber",
+          "PLIdentityCard",
+          "PLNationalID",
+          "PLNationalIDV2",
+          "PLPassportNumber",
+          "PLTaxIdentificationNumber",
+          "PLREGONNumber",
+          "PTCitizenCardNumber",
+          "PTCitizenCardNumberV2",
+          "PTTaxIdentificationNumber",
+          "ROPersonalNumericalCode",
+          "RUPassportNumberDomestic",
+          "RUPassportNumberInternational",
+          "SANationalID",
+          "SGNationalRegistrationIdentityCardNumber",
+          "SKPersonalNumber",
+          "SITaxIdentificationNumber",
+          "SIUniqueMasterCitizenNumber",
+          "ZAIdentificationNumber",
+          "KRResidentRegistrationNumber",
+          "ESDNI",
+          "ESSocialSecurityNumber",
+          "ESTaxIdentificationNumber",
+          "SQLServerConnectionString",
+          "SENationalID",
+          "SENationalIDV2",
+          "SEPassportNumber",
+          "SETaxIdentificationNumber",
+          "SWIFTCode",
+          "CHSocialSecurityNumber",
+          "TWNationalID",
+          "TWPassportNumber",
+          "TWResidentCertificate",
+          "THPopulationIdentificationCode",
+          "TRNationalIdentificationNumber",
+          "UKDriversLicenseNumber",
+          "UKElectoralRollNumber",
+          "UKNationalHealthNumber",
+          "UKNationalInsuranceNumber",
+          "UKUniqueTaxpayerNumber",
+          "USUKPassportNumber",
+          "USBankAccountNumber",
+          "USDriversLicenseNumber",
+          "USIndividualTaxpayerIdentification",
+          "USSocialSecurityNumber",
+          "UAPassportNumberDomestic",
+          "UAPassportNumberInternational",
+          "Organization",
+          "Email",
+          "URL",
+          "Age",
+          "PhoneNumber",
+          "IPAddress",
+          "Date",
+          "Person",
+          "Address",
+          "All",
+          "Default"
+        ]
       },
       "name": "piiCategories",
-      "default": "Default",
+      "x-ms-client-default": "Default",
```

Comment:
@abhahn  this doesn't have to be ["Default"] due to type: array? Or is this basically specifying that out of the enums, use this as the default value in the array?

Reply:
Yeah, to be completely honest I don't really understand why this works but setting `"default": ["Default"]` doesn't.  This is the only way that I could get both Swagger model validation and Swagger linting to pass.  I think it may have the consequence of leaving the endpoint with no default value from the standpoint of APIM, which may be a problem.  If I understand correctly, I think `x-ms-client-default` will result in the generated SDK methods having a default value.  

Of the things I tried, here are the combinations that will fail one step or the other:  `"default": ["Default"]` (fails linting), `"default": "Default"` (fails model validation), and `"x-ms-client-default": ["Default"]` (fails linting).

I think that `"default": ["Default"]` *should* be the correct way (even though it is failing) because it is explicitly shown as the way to set default values for enums in the [official OpenApi 2.0 docs](https://swagger.io/docs/specification/2-0/describing-parameters/).  Unfortunately this page doesn't have anchors, but if you Ctrl+F you can find the following text for the example:  "You can also specify the default array that the server will use if this parameter is omitted"

Reply:
good catch, I didn't see the bit in the doc that allows you to omit the brackets

Reply:
@tjprescott , just wondering if you could help us to understand why using an array to represent the default value using the `default` property won't work here.  I'd like to resolve this as soon as possible so we can merge the main PR for v3.1-preview.4.

Reply:
@abhahn and I discussed this offline. 

Given the property/parameter is an array type, like Abby pointed the swagger should represent the Default in an array format. But today only the property/parameter with schemas - string, integer, long, float, double, boolean is supported.

We kicked off a discussion with the AutoRest core team to evaluate enabling array schema for `x-ms-client-default` extension.

It may take some time for AutoRest core to evaluate and decide on the this feature request.

Meantime, we should not use this extension for array cases, as it cannot be translated to meaningful representation in autorest or any languages.

Reply:
thanks! Laurent @lmazuel for all offline help on this.